src/renderer/store/TimelineSpace/Contents/Local.ts

```diff
@@ -0,0 +1,133 @@
+import Mastodon, { Status, Response } from 'megalodon'
+import { Module, MutationTree, ActionTree } from 'vuex'
+import { RootState } from '@/store'
+
+export interface LocalState {
+  timeline: Array<Status>,
+  unreadTimeline: Array<Status>,
+  lazyLoading: boolean,
+  heading: boolean,
+  filter: string
+}
+
+const state = (): LocalState => ({
+  timeline: [],
+  unreadTimeline: [],
+  lazyLoading: false,
+  heading: true,
+  filter: ''
+})
+
+export const MUTATION_TYPES = {
+  CHANGE_HEADING: 'changeHeading',
+  APPEND_TIMELINE: 'appendTimeline',
+  UPDATE_TIMELINE: 'updateTimeline',
+  MERGE_TIMELINE:' mergeTimeline',
```

Comment:
__[ESLint]__ Error - Missing space before value for key 'MERGE_TIMELINE'. (key-spacing) ([link](https://sider.review/gh/repos/124234495/pulls/876#issue-3979045))

<sub>posted by Sider</sub>